### ISSUE TITLE
replace CAM-SE var-res control volume grids with newer/better grids

### ISSUE THREAD

**Issue by adamrher** [ESCOMP/CAM#665]
### What is the feature/what would you like to discuss?

Replace ESMF mesh files for ARCTIC, ARCTICGRIS and CONUS grids in ccs_config, using an improved algorithm. By improved, I mean the control volumes are more regular, with less frequent occurrences of non-convex polygons (which tends to reduce the accuracy of the mapping ... although ESMF can handle these fine). This is achieved by only allowing up to 6 sided polygons, as opposed to the current algorithm that permits 14 sided polygons. I'm referring to this new algorithm as "chevrons" since it does tend to produce some non-convex 6-sided chevrons.

Changing the mesh files will change answers for anything that uses mapping weights. But it's not clear whether the F-compsets will have answer changes because nothing is mapped -- everything is on the same grid.

I've verified that the new grids aren't of a lower quality by checking their conservation properties (mapping them to themselves using ESMF and then checking those maps) compared to the current ones. The global area has smaller errors, and the relative/maximum/least-squares errors are all reduced using these new mesh files.

```
/glade/work/aherring/grids/var-res/ne0np4.ARCTIC.ne30x4/grids/ne0ARCTICne30x4_esmf_chevrons_c220827.nc
/glade/work/aherring/grids/var-res/ne0np4.ARCTICGRIS.ne30x8/grids/ne0ARCTICGRISne30x8_esmf_chevrons_c220928.nc
```

These will need to be added to the inputdata and paths updated in ```ccs_config/component_grids_nuopc.xml```. I don't think we should update the MCT mapping files, as MCT will be deprecated soon/eventually.

[the CONUS mesh file is still processing. Will update with path when it completes].

### Is there anyone in particular you want to be part of this conversation?

@PeterHjortLauritzen 

### Will this change (regression test) answers?

Yes

### Will you be implementing this enhancement yourself?

No

**Comment by jedwards4b** [comment]
I don't believe we need the scripgrids in inputdata.

**Comment by adamrher** [comment]
I've opened an issue in ccs_config.

**Comment by adamrher** [comment]
I'm thinking that I should delete this issue so we aren't having two conversations. And move the primary conversation to the ccs_config issue. Is that OK?

**Comment by cacraigucar** [comment]
I agree that the discussion should happen in the ccs_config_cesm repo.
